### ISSUE TITLE
Feature request: colorize badges for measures that are part of the quality gate

### ISSUE THREAD

**Issue by ganncamp** [QualInsight/qualinsight-plugins-sonarqube-badges#28]
ideally:
- Red for values causing QG failure
- Orange for values in the Warning range of a QG _whether the QG is in Warning or Error state_
- Green _or_ grey for passing values that are included in the QG _whether or not the QG is in Warning or error state_

**Comment by pawlakm** [comment]
Implementation details:
- prerequisites: `2.0.1` needs to be released
- in `MeasuresBadgeRequestHandler.handle()` call `api/qualitygates/project_status` with `projectKey` parameter in order to retrieve the QG for the project.
- parse `conditions`
- for each condition, check if `metric`passed to the measure badge WS is equal to `metricKey`. 
- if yes, after having built a `MeasureHolder` set its background color using `MeasureHolder.setBackgroundColor(SVGImageColor)` based on `status` value (OK -> `GREEN`, KO -> `RED`, WARN -> `ORANGE`)
- else do nothing (defaults to `GRAY`)

Out of scope (another issues need to be created in order to handle these) :
- add cache for generated badges (cache strategy needs to be defined: by project, platform based, TTL, etc.)
- add cache for QG status data (call the WS only if the QG status for the project hasn't been refreshed for X minutes)
- share QG status cache with QG badges WS

**Comment by pawlakm** [comment]
fixed in branch colorise-measures-badges
